utils/logger/logger.go

```diff
@@ -43,7 +44,7 @@ func Setup() {
 // Most verbose logging level.
 func Debug(args ...interface{}) (int, error) {
 	if levelDebug >= level {
-		fmt.Print("DEBUG: ")
+		fmt.Print(time.Now().Format(time.RFC850), " DEBUG: ")
```

Comment:
We already get timestamps from Cloudwatch logs and also syslog service. These would be extra fields in the final logs.